pkg/platform/factory/factory.go

```diff
@@ -146,7 +146,7 @@ func getContainerBuilderConfiguration(platformConfiguration interface{}) *contai
 	}
 	if containerBuilderConfiguration.KanikoImage == "" {
 		containerBuilderConfiguration.KanikoImage = getEnvOrDefault("NUCLIO_KANIKO_CONTAINER_IMAGE",
-			"gcr.io/kaniko-project/executor:v0.9.0")
+			"gcr.io/kaniko-project/executor:v0.13.0")
```

Comment:
Please don't forget to add this to artifact version manifest so it will be packaged